rspace/src/main/scala/coop/rchain/rspace/IReplaySpace.scala

```diff
@@ -18,6 +18,13 @@ trait IReplaySpace[F[_], C, P, A, R, K] extends ISpace[F, C, P, A, R, K] {
     *  @param log A [Log] with permitted operations
     */
   def resetAndRig(startRoot: Blake2b256Hash, log: trace.Log)(implicit syncF: Sync[F]): F[Unit] =
+    rig(log)
+      .flatMap { _ =>
+        // reset to the starting checkpoint
+        reset(startRoot)
+      }
```

Comment:
Is this the same as 

`rig >> reset`?

Maybe the function should be called rigAndReset

Reply:
Good catch!